src/python/national_voter_file/us_states/de/transformer.py

```diff
@@ -0,0 +1,279 @@
+import csv
+import os
+import re
+import sys
+from datetime import date
+
+from national_voter_file.transformers.base import (DATA_DIR,
+                                                   BasePreparer,
+                                                   BaseTransformer)
+import usaddress
+
+__all__ = ['default_file', 'StatePreparer', 'StateTransformer']
+
+default_file = 'DE_Sample.csv'
+
+class StatePreparer(BasePreparer):
+    state_path = 'de' # Two letter code for state
+    state_name='Delaware' # Name of state with no spaces. Use CamelCase
```

Comment:
spaces around = and other stylistic things were caught by travis. https://travis-ci.org/national-voter-file/national-voter-file/jobs/202202474